documentation/release-process.md

```diff
@@ -0,0 +1,56 @@
+# Release Process Documentation
+
+Based on https://github.com/dotnet/sdk/issues/41607
+
+## Ideal Process
+
+**Note:** This process may vary based on the workload owners.
+**Note:** Most of this process today is done through disparate builds in each workload repo and this repo (or manually). Below is outlining our desired goal.
```

Comment:
These aren't rendered on separate lines.  You can also use the actual Note notation:
```suggestion
> [!NOTE]
> This process may vary based on the workload owners.

> [!NOTE]
> Most of this process today is done through disparate builds in each workload repo and this repo (or manually). Below is outlining our desired goal.
```